python/python/raphtory/graphqlserver.py

```diff
@@ -41,38 +42,44 @@ def wait_for_online(self):
         Waits for the server to be online. This is called automatically when run_server is called.
         """
         while True:
-             try:
-                r = requests.get("http://localhost:"+str(self.port))
+            try:
+                r = requests.get("http://localhost:" + str(self.port))
                 if r.status_code == 200:
                     return True
-             except:
+            except:
                 pass
-             time.sleep(1)
+            time.sleep(1)
+
+
+async def __from_map_and_directory(graphs, graph_dir, port):
+    await internal_graphql.from_map_and_directory(graphs, graph_dir, port)
+
 
-async def __from_map_and_directory(graphs,graph_dir,port):
-    await internal_graphql.from_map_and_directory(graphs,graph_dir,port)
+async def __from_directory(graph_dir, port):
+    await internal_graphql.from_directory(graph_dir, port)
```

Comment:
did we really want `__` here? They are special in python and cause weird name mangling. If you just want to denote this as a private method you should use `_`